virridy/non_targets_workflow/00_virridy_preparatory_workflow_old.Rmd

```diff
@@ -154,10 +159,6 @@ threshold_lookup <- all_data_summary_stats_list %>%
   dplyr::bind_rows()
 
 readr::write_csv(threshold_lookup, 'data/qaqc/seasonal_thresholds_virridy.csv')
-
-# also save the threshold lookup table as a RDS 
-# saveRDS(threshold_lookup, 'data/qaqc/seasonal_thresholds_virridy.RDS')
-
 ```
 
 *Compare seasonal thresholds to our own "reasonable" ranges for parameters generally:*
```

Comment:
Question: This does not need to be part of the FC system right?

Reply:
correct! this will not be added to FC